GitDepend/CommandLine/Options.cs

```diff
@@ -48,7 +54,12 @@ private Options()
         [HelpOption]
         public string GetUsage()
         {
-            return HelpText.AutoBuild(this, (HelpText current) => HelpText.DefaultParsingErrorsHandler(this, current));
+            var versionUpdateChecker = DependencyInjection.Resolve<IVersionUpdateChecker>();
+
+            var appendString = versionUpdateChecker.CheckVersion("gitdepend", "kjjuno");
+            appendString += versionUpdateChecker.CheckVersion("choco", "chocolatey");
```

Comment:
we should the reference to choco. We don't need to be reporting on their version. If we want to add reporting for the chocolatey version later we can.